components/kyma-environment-broker/internal/cls/client_test.go

```diff
@@ -130,6 +110,58 @@ func TestCreateInstance(t *testing.T) {
 	}
 }
 
+func TestCreateBinding(t *testing.T) {
+	smClientMock := &automock.Client{}
+	creds := make(map[string]interface{})
+	creds["Kibana-endpoint"] = "kibUrl"
+	creds["Fluentd-username"] = "fbUser"
+	creds["Fluentd-password"] = "fbPass"
+	creds["Fluentd-endpoint"] = "fbEndpoint"
+	resB := servicemanager.BindingResponse{
+		Binding:      servicemanager.Binding{Credentials: creds},
+		HTTPResponse: servicemanager.HTTPResponse{StatusCode: 200},
+	}
+	config := getConfig()
+	smClientMock.On("Bind", mock.Anything, mock.Anything, mock.Anything, mock.Anything).Return(&resB, nil)
+	sut := NewClient(config, logrus.New())
+
+	br := BindingRequest{
+		InstanceKey: servicemanager.InstanceKey{},
+		BindingID:   "",
+	}
+	expectedClsOverrides := &ClsOverrides{
+		FluentdEndPoint: "fbEndpoint",
+		FluentdPassword: "fbPass",
+		FluentdUsername: "fbUser",
+		KibanaUrl:       "kibUrl",
+	}
+	res, err := sut.CreateBinding(smClientMock, &br)
+	require.NoError(t, err)
+	require.Equal(t, expectedClsOverrides, res)
+}
+
+func getConfig() *Config {
```

Comment:
We don't need a function for this. Instead, we can use a global variable.

Reply:
done